src/documentation/literal.rs

```diff
@@ -31,9 +31,17 @@ impl Default for CommentVariant {
 impl ToString for CommentVariant {
     fn to_string(&self) -> String {
         match self {
-            CommentVariant::TripleSlash => "/// ".into(),
-            CommentVariant::DoubleSlashEM => "//! ".into(),
-            CommentVariant::MacroDocEq => "".into(),
+            CommentVariant::TripleSlash => "///".into(),
+            CommentVariant::DoubleSlashEM => "//!".into(),
+            CommentVariant::MacroDocEq(p) => {
+                let raw = match p {
+                    0 => "".to_string(),
```

Comment:
I am generally a bit suspicious when using `.to_string()` but as a matter of fact to it calls `.to_owned()` internally https://doc.rust-lang.org/src/alloc/str.rs.html#204-206 https://doc.rust-lang.org/src/alloc/string.rs.html#2272-2277

So all my comments before were kind of pointless here, except for saving a few chars :man_shrugging:

Reply:
what do you suggest?  Try to drop all `to_string()` and use `&str` instead?

Reply:
Mostly informational :) code is fine as is